workspace-manager-clienttests/src/main/java/scripts/testscripts/GetWorkspace.java

```diff
@@ -0,0 +1,87 @@
+package scripts.testscripts;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.client.ApiClient;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.client.ApiException;
+import bio.terra.workspace.model.CreateWorkspaceRequestBody;
+import bio.terra.workspace.model.CreatedWorkspace;
+import bio.terra.workspace.model.WorkspaceDescription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.utils.WorkspaceManagerServiceUtils;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class GetWorkspace extends TestScript {
+    private static final Logger logger = LoggerFactory.getLogger(GetWorkspace.class);
+    private UUID id;
+    private CreatedWorkspace workspace;
+
+    @Override
+    public void setup(List<TestUserSpecification> testUsers) throws Exception {
+        assertThat("There must be at least one test user in configs/testusers directory.", testUsers!=null && testUsers.size()>0);
+        id = UUID.randomUUID();
+        ApiClient apiClient = WorkspaceManagerServiceUtils.getClientForTestUser(testUsers.get(0), server);
+        WorkspaceApi workspaceApi = new WorkspaceApi(apiClient);
+        try {
+            CreateWorkspaceRequestBody requestBody = new CreateWorkspaceRequestBody();
+            requestBody.setId(id);
+            requestBody.setJobId("testrunner");
+            requestBody.setSpendProfile("testrunner");
```

Comment:
if you run in to problems using the string "testrunner" for a spend profile, just leave it out; it's not required

Reply:
Set spend profile to null.